components/dashboard/src/settings/Preferences.tsx

```diff
@@ -55,7 +55,7 @@ export default function Preferences() {
                         <div className="flex justify-center mt-3">
                             <img className="w-16 filter-grayscale self-center" src={vscode} />
                         </div>
-                        <span className="mt-2 ml-2 self-center rounded-xl py-0.5 px-2 text-sm bg-orange-100 text-orange-700 dark:bg-orange-600 dark:text-orange-100 font-semibold">LATEST</span>
+                        <span className="mt-2 ml-2 self-center rounded-xl py-0.5 px-2 text-sm bg-orange-100 text-orange-700 dark:bg-orange-600 dark:text-orange-100 font-semibold">INSIDER</span>
```

Comment:
```suggestion
                        <span className="mt-2 ml-2 self-center rounded-xl py-0.5 px-2 text-sm bg-orange-100 text-orange-700 dark:bg-orange-600 dark:text-orange-100 font-semibold">INSIDERS</span>
```

I think it is intended to be `INSIDERS` - https://code.visualstudio.com/insiders

![image](https://user-images.githubusercontent.com/69431456/135585769-3fcedfb0-5aee-4fa7-94f7-cce5d2c0d948.png)

Reply:
Oooh! @filiptronicek already suggested it ❤️